jerry-ext/external/external-register.c

```diff
@@ -0,0 +1,43 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "jerryscript-ext/external.h"
+
+
+/**
+ * Register a JavaScript function in the global object.
+ *
+ * Note:
+ *      returned value must be freed with jerry_release_value, when it is no longer needed.
+ *
+ * @return true value - if the operation was successful
+ *         value marked with error flag - otherwise
+ */
+jerry_value_t
+jerryx_external_register_global_function (const jerry_char_t *name_p, /**< name of the function */
```

Comment:
We should really write this down at some point of time so that we remember what is what in our chosen naming scheme. In #1740, when jerry-ext vs jerry-util was disputed (for the name of the library), we have decided for ext, to stand for extension (https://github.com/jerryscript-project/jerryscript/pull/1740#issuecomment-295262356, https://github.com/jerryscript-project/jerryscript/pull/1740#issuecomment-295266702, https://github.com/jerryscript-project/jerryscript/pull/1740#issuecomment-295920812). And identifiers of the jerry-ext library to have jerryx_ prefix (where x still stands for extension).